io.github.androidvirtualpen.virtualpen.json

```diff
@@ -0,0 +1,49 @@
+{
+    "id": "io.github.androidvirtualpen.virtualpen",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "6.7",
+    "sdk": "org.kde.Sdk",
+    "command": "virtual-pen-linux-host",
+    "finish-args": [
+	"--socket=wayland",
+	"--socket=fallback-x11",
+	"--share=ipc",
+	"--filesystem=home",
+	"--device=all",
+	"--share=network"
+    ],    
+    "modules":[
+	{	    
+	    "name": "virtualpenlinuxhost",
+	    "buildsystem": "simple",
+	    "build-commands": [
+		"install -Dvm 755 virtual-pen-linux-host -t /app/bin/",
+		"install -Dvm 644 libusb-1.0.so.0 -t /app/lib/",
```

Comment:
Why aren't you building this? This does not provide the library for both arches.

Reply:
I didn't want to go through the trouble of setting up a build env for both my app and libusb. I'll give it a shot but meanwhile, I'm very much tempted to restrict the app to x86_amd64.

Reply:
building from source is always prefered.

libusb is easy to build just use the `shared-modules`. https://github.com/flathub/shared-modules/

Reply:
Definitely will give it a try. Hopefully, not too many hours will be spent on that.